Source/Dafny/Compiler-java.cs

```diff
@@ -2923,58 +2932,63 @@ private class GenericArrayElementLvalue : ILvalue {
         wr.Write("public static <{1}> Tuple{0}<{1}>", i, Util.Comma(i, j => $"T{j}"));
       }
       wr.Write(" create({0})", Util.Comma(i, j => $"T{j} _{j}"));
-      using (var w = wr.NewBlock("")) {
+      {
+        var w = wr.NewBlock(""); 
         w.WriteLine("return new Tuple{0}({1});", i, Util.Comma(i, j => $"_{j}"));
       }
 
       wr.WriteLine();
       wr.WriteLine("@Override");
-      using (var wrEquals = wr.NewBlock("public boolean equals(Object obj)")) {
-        wrEquals.WriteLine("if (this == obj) return true;");
-        wrEquals.WriteLine("if (obj == null) return false;");
-        wrEquals.WriteLine("if (getClass() != obj.getClass()) return false;");
-        wrEquals.WriteLine($"Tuple{i} o = (Tuple{i}) obj;");
-        if (i != 0) {
-          wrEquals.WriteLine("return {0};", Util.Comma(" && ", i, j => $"java.util.Objects.equals(this._{j}, o._{j})"));
-        } else {
-          wrEquals.WriteLine("return true;");
-        }
+      var wrEquals = wr.NewBlock("public boolean equals(Object obj)");
+      wrEquals.WriteLine("if (this == obj) return true;");
+      wrEquals.WriteLine("if (obj == null) return false;");
+      wrEquals.WriteLine("if (getClass() != obj.getClass()) return false;");
+      wrEquals.WriteLine($"Tuple{i} o = (Tuple{i}) obj;");
+      if (i != 0) {
+        wrEquals.WriteLine("return {0};", Util.Comma(" && ", i, j => $"java.util.Objects.equals(this._{j}, o._{j})"));
+      }
+      else {
```

Comment:
Move this `else` to the previous line

Reply:
I haven't found a way to configure ReSharper and/or Rider to keep the else on the same line as the `}`. Do you use any of those tools?

Reply:
FWIW, in Rider, it might be some combination of Preferences->Editor->Code Style->C#->Braces Layout->Other = "At end of line (K&R style)" or Preferences->Editor->Code Style->C#->Line Breaks and Wrapping->Arrangement of Statements->Place "else" on new line.

Reply:
Found it, thanks.

Reply:
There's some way in Rider to save these settings on a project basis. I think I have that turned on locally, but I don't know what file it writes it to. It would be nice to check in such a style file into the project.